src/index.js

```diff
@@ -7,17 +7,11 @@ import 'primereact/resources/primereact.min.css';
 import 'primeicons/primeicons.css';
 
 import App from './App';
-import { Provider } from 'react-redux'
-
 import * as serviceWorker from './serviceWorker';
 
-import store from './store';
-
 ReactDOM.render(
-    <Provider store={store}>
-        <App />
-    </Provider>
-    , document.getElementById('root'));
+ <App />
+   , document.getElementById('root'));
```

Comment:
You need a `<Provider store={store}>... </Provider>` or your `<App />` will not work properly.

Reply:
The Provider is in App.js and seems to be working fine.

Reply:
Moving provider back to index.js...